test-vectors/tx-out-records/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "mc-test-vectors-tx-out-records"
+version = "1.3.0-pre0"
+authors = ["MobileCoin"]
+edition = "2018"
+
+[dependencies]
+hex = "0.4"
+mc-test-vectors-definitions = { path = "../definitions" }
```

Comment:
Nit: Separate MC and external deps, throughout this file.

Reply:
I had it that way before, but I think it was giving me issues with the linter's "sorted dependency" requirement. Regardless, I prefer it to be this way because when I want to find a dependency, it's easier if the dependencies exist in one sorted block. If there are multiple blocks, then I sometimes have trouble quickly locating the dependency, and have to resort to ctrl - F.